gprofiler/main.py

```diff
@@ -943,7 +980,8 @@ def main() -> None:
         logger.info(
             "Running gProfiler", version=__version__, commandline=" ".join(sys.argv[1:]), arguments=args.__dict__
         )
-
+        if processes_to_profile is not None:
+            logger.info("Target PIDs given by --pids: ", pids=[process.pid for process in processes_to_profile])
```

Comment:
This looks like:
```
[2023-04-12 15:56:36,138] INFO: gprofiler: Target PIDs given by --pids:  (pids=[1])
```

No need for the `:` and the space, since logging adds it anyway when formatting the extras.